libstuff/libstuff.cpp

```diff
@@ -2495,25 +2497,6 @@ void SQueryLogClose() {
     }
 }
 
-// --------------------------------------------------------------------------
-// Called by SQLite in response to query
```

Comment:
Per the sqlite team's suggestion, we replace `sqlite3_exec` with the individual component calls that make it up, for more accurate timing, so we don't need this.